src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: $day-font;
```

Comment:
The variable `$day-font` is used here, but it is not defined in this file. Make sure it is defined in a file that is imported before this one, or define it here.